crates/fuel-gas-price-algorithm/gas-price-analysis/src/charts.rs

```diff
@@ -267,16 +267,14 @@ pub fn draw_profit(
     const ACTUAL_PROFIT_COLOR: RGBColor = BLACK;
     const PROJECTED_PROFIT_COLOR: RGBColor = RED;
     const PESSIMISTIC_BLOCK_COST_COLOR: RGBColor = BLUE;
-    let actual_profit_gwei: Vec<_> = actual_profit
-        .into_iter()
-        .map(|x| x / ONE_GWEI as i128)
-        .collect();
+    let actual_profit_gwei: Vec<_> =
+        actual_profit.iter().map(|x| x / ONE_GWEI as i128).collect();
```

Comment:
While unrelated to this PR, can we use `i28::from` here instead of an `as` conversion to show that the conversion is infallible (which I hope it is 😅)

Reply:
I've made the change. I think in general the analysis code is a little messier, but now that we're considering keeping it around and maybe expanding it's role it's probably best we get more strict with it.

Reply:
Ah right, didn't notice this was in the analysis code :sweat_smile: thanks anyway!